tests/test_aprsd_webchat_extension.py

```
@@ -99,3 +99,102 @@ def test_process_our_message_packet(
+
+class TestSendMessageNamespace(unittest.TestCase):
+    """Tests for WebSocket GPS beacon handling with symbol parameter."""
+
+    def config_and_init(self):
+        CONF.callsign = fake.FAKE_TO_CALLSIGN
+        CONF.trace_enabled = False
+
+    @mock.patch("aprsd_webchat_extension.cmds.webchat.tx")
+    @mock.patch("aprsd_webchat_extension.cmds.webchat.socketio")
+    @mock.patch("aprsd_webchat_extension.cmds.webchat.packets")
+    def test_on_gps_with_symbol(self, mock_packets, mock_socketio, mock_tx):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_tx`

(ARG002)